packages/flame/lib/src/experimental/column_component.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flame/src/experimental/layout_component.dart';
+import 'package:flutter/widgets.dart';
+import 'package:vector_math/vector_math_64.dart';
+
+/// Allows laying out children in a column by defining a [MainAxisAlignment]
+/// type.
+class ColumnComponent extends LayoutComponent {
+  ColumnComponent({
+    MainAxisAlignment mainAxisAlignment = MainAxisAlignment.start,
```

Comment:
Perhaps call this simply `alignment`?

Reply:
What if we would like to add a `crossAxisAlignment` later?

Reply:
then we could call it `crossAlignment`

Reply:
It's pretty nice to keep the same naming as for the rows and columns for Flutter though, even though their naming isn't optimal.

Reply:
It is superfluous though: when you say `alignment: MainAxisAlignment.center`, you're already mentioning that this is a main axis alignment. Saying it twice sounds tautological.

Also, Flutter has `mainAxisSize`, whereas we use simple `size`.

Reply:
Yeah, I agree it's a bad name. I still think I lean to having it consistent with Flutter though.

`mainAxisSize` is not the same thing as our `size`.

Reply:
Let's go with the better name (`alignment`) and skip being consistent with Flutter, I think our users will easily be able to figure it out anyways.